website/content/en/docs/faq/colima.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Lima (underlying technology)
+weight: 0
+---
+
+## "How does Colima relate to Lima?"
+
+Colima is built on top of [Lima](https://github.com/lima-vm/lima), which provides Linux virtual machines on macOS and Linux with automatic file sharing and port forwarding.
+
+Lima is the underlying virtualization technology that makes Colima possible.
+Colima provides a simplified interface specifically focused on container runtimes (Docker, containerd) and Kubernetes.
+
+| Feature            | Lima                              | Colima                              |
+|----------------------|-----------------------------------|-------------------------------------|
+| containerd           | `limactl start`                   | `colima start --runtime=containerd` |
+| Docker               | `limactl start template://docker` | `colima start`                      |
+| Kubernetes (k3s)     | `limactl start template://k3s`    | `colima start --kubernetes`         |
+
+## Key advantages of Colima over raw Lima
+
+- **Simplified CLI**: Single `colima start` command vs complex Lima templates
```

Comment:
Lima has a simple `limactl start` CLI too, and Colima has complex YAML templates too